pkgs/tools/inputmethods/keyd/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, stdenv }:
+
+stdenv.mkDerivation rec {
+
+  pname = "keyd";
+  version = "2.2.5-beta";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Mb5LU8xsX8H/k3yMQiaCw4BVMphxNo25HPE0sMWTD8Y=";
+  };
+  installPhase = ''
+    make install DESTDIR=$out PREFIX=
+  '';
+
```

```suggestion
  makeFlags = [ "DESTDIR=${placeholder "out"}" "PREFIX=" ];
```
This should also do it, haven't tested it but that's the idea

---

I think it's the other way around (set prefix, unset deskdir)